feature-libs/product/configurators/common/src/components/config-form/config-form.component.ts

```diff
@@ -74,28 +74,19 @@ export class ConfigFormComponent implements OnInit {
       event.changedAttribute
     );
 
-    // Wait until update is triggered first, then wait until update is finished, to be sure that the configuration
-    // is changed before the group status is set. This cannot be done in the effects, as we need to call the facade layer.
+    // Wait until update is done until setting the group status
     this.configuratorCommonsService
       .isConfigurationLoading(owner)
       .pipe(
-        filter((isLoading) => isLoading.valueOf()),
+        filter((isLoading) => !isLoading),
```

Comment:
To be extra safe here, I'd add `distinctUntilChanged` operator before the `filter`. In case the `isConfigurationLoading()` fires multiple times with the same value (shouldn't happen, but just in case).

Reply:
Done